_pages/about-us/teams/products-and-platforms.md

```diff
@@ -51,10 +53,10 @@ If you're a P&P project lead, expect to:
 
 * Report weekly on your project's status, blockers, and financial status via the weekly meeting agenda.
 
-* Be involved in the staffing process
+* Be involved in the staffing process.
```

Comment:
@abrouilette That works for me 👍 Perhaps we could change the action item to "Be involved in the staffing process, which is outlined in full here [link]," or something similar.